linchpin/linchpin.conf

```diff
@@ -88,3 +89,23 @@ level = logging.INFO
 
 [ansible]
 console = True
+
+[playbook_pre_states]
+up = preup
+destroy = predestroy
+
+[playbook_post_states]
+up = postup
+destroy = postdestroy
+
+[states]
+# in future each state will carry a comma seperated substates
+preup = 'preup'  
+predestroy = 'predestroy'
+postup = 'postup'
+postdestroy = 'postdestroy'
+
+[extns]
+resource = '.output'
+inventory = '.inventory'
```

Comment:
I think @samvarankashyap might be used to the fact that yaml required quotes around things while the ini style configuration does not. Additionally, the ini configuration functionality supports interpolation with ConfigParser, which is great to be able to reuse the variables in another variable.